modules/kit-elements-defaults/utils/settings-sanitizer.php

```diff
@@ -209,4 +218,37 @@ private function run_import_export_sanitize_process( Document $document, $proces
 
 		return $this;
 	}
+
+	/**
+	 * Get all the available settings of a specific element, including responsive settings.
+	 *
+	 * @param array $controls
+	 *
+	 * @return array
+	 */
+	private function get_avaliable_settings( array $controls ) {
+		$control_keys = array_keys( $controls );
+
+		$optional_responsive_keys = [
+			Breakpoints_Manager::BREAKPOINT_KEY_MOBILE,
+			Breakpoints_Manager::BREAKPOINT_KEY_MOBILE_EXTRA,
+			Breakpoints_Manager::BREAKPOINT_KEY_TABLET,
+			Breakpoints_Manager::BREAKPOINT_KEY_TABLET_EXTRA,
+			Breakpoints_Manager::BREAKPOINT_KEY_LAPTOP,
+			Breakpoints_Manager::BREAKPOINT_KEY_WIDESCREEN,
+		];
+
+		$settings = [];
+
+		foreach ( $control_keys as $control_key ) {
+			// Add the responsive settings.
+			foreach ( $optional_responsive_keys as $responsive_key ) {
```

Comment:
Not sure that I'm getting what's going on here, how do you know which controls are responsive?

Reply:
I am assuming all the controls can be responsive, so I add the suffix for each control.